### PR TITLE
docs: OPS-4863

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR adds new content pages for the CAPI Image Builder utility and reorganizes the BYOOS section.

## Changed Pages

💻 [BYOOS](https://deploy-preview-3523--docs-spectrocloud.netlify.app/byoos/)
💻 [CAPI Image Builder](https://deploy-preview-3523--docs-spectrocloud.netlify.app/byoos/capi-image-builder/)
💻 [Build Image for VMware vSphere](https://deploy-preview-3523--docs-spectrocloud.netlify.app/byoos/capi-image-builder/build-image-vmware/)
💻 [RHEL and Palette eXtended Kubernetes](https://deploy-preview-3523--docs-spectrocloud.netlify.app/byoos/capi-image-builder/build-image-vmware/rhel-capi/)
💻 [Rocky and Palette eXtended Kubernetes](https://deploy-preview-3523--docs-spectrocloud.netlify.app/byoos/capi-image-builder/build-image-vmware/rocky-capi/)
💻 [Configuration Reference](https://deploy-preview-3523--docs-spectrocloud.netlify.app/byoos/capi-image-builder/config-reference/)
💻 [CAPI Image Builder Compatibility Matrix](https://deploy-preview-3523--docs-spectrocloud.netlify.app/byoos/capi-image-builder/comp-matrix-capi-builder/)

## Jira Tickets

🎫 [OPS-4863](https://spectrocloud.atlassian.net/browse/OPS-4863)

## Backports

Can this PR be backported?

- [ ] Yes. _Remember to add the relevant backport labels to your PR._
- [x] No. _Please leave a short comment below about why this PR cannot be backported._
- [ ] 
Release branch.

[OPS-4863]: https://spectrocloud.atlassian.net/browse/OPS-4863?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/byoos/capi-image-builder/capi-image-builder.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+sidebar_position: 0
+tags: ["operating system", "byoos", "capi iamge builder"]
+---
+
+The CAPI Image Builder is a tool we developed and maintain to reduce the challenges associated with creating images for
+Kubernetes clusters. It is based on the upstream
+[Kubernetes Image Builder (KIB)](https://image-builder.sigs.k8s.io/introduction.html) project.
+
+The tool includes all the dependencies required to build FIPS and non-FIPS images within a Docker container, eliminating
+the need to install any dependencies on your local machine. This is particularly useful when working in air-gapped
```
Comment (lennessyy) [inline docs/docs-content/byoos/capi-image-builder/capi-image-builder.md:16]:
```suggestion
the need to install any dependencies on your local machine. This is particularly useful when working in airgapped
```

We use airgap as a single word. I used to write like this when i didn't know but we've gone and corrected all of them

File: docs/docs-content/byoos/image-builder/image-builder.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+[Azure VM Image Builder](https://azure.microsoft.com/en-us/products/image-builder) for Azure. You can also use platform
+agnostic tools, such as [HashiCorp Packer](https://developer.hashicorp.com/packer), or something more tailored to
+Kubernetes, such as the [Kubernetes Image Builder](https://image-builder.sigs.k8s.io/introduction.html) (KIB) project.
+
+## Kubernetes Image Builder
+
+KIB is a project designed to help users create images for various platforms. The project is a consolidation of multiple
+tools that together work to create an artifact, or in simpler terms, a custom image.
+
+You can use the custom images created by KIB with Palette, assuming the infrastructure provider is supported in Palette.
```
Comment (lennessyy) [inline docs/docs-content/byoos/image-builder/image-builder.md:23]:
```suggestion
You can use the custom images created by KIB with Palette, assuming the infrastructure provider is supported by Palette.
```

**Review by lennessyy — APPROVED** [review]

File: docs/docs-content/byoos/capi-image-builder/capi-image-builder.md (hunk 1 above)
Comment (caroldelwing) [reply docs/docs-content/byoos/capi-image-builder/capi-image-builder.md:16]: Omg, I looked at our word list and still wrote it wrong 😶‍🌫️
